fe/fe-core/src/main/java/org/apache/doris/nereids/memo/GroupExpression.java

```diff
@@ -276,7 +276,23 @@ public void mergeToNotOwnerRemove(GroupExpression target) {
         this.getLowestCostTable()
                 .forEach((properties, pair) -> target.updateLowestCostTable(properties, pair.second, pair.first));
         // requestPropertiesMap
-        target.requestPropertiesMap.putAll(this.requestPropertiesMap);
+        for (Map.Entry<PhysicalProperties, PhysicalProperties> entry : requestPropertiesMap.entrySet()) {
```

Comment:
1）necessary to wrap it as a function, e.g, updateRequestPropertiesMap, similar with updateLowestCostTable?
2) make sure it is the only invoking point which need to sync LowestCostTable and RequestPropertiesMap during the whole group merging?

Reply:
we have two point to update them. the other one is when we calculate cost and enforcer. there are no common code between the two points, because we could simply only update requestPropertiesMap when do updateLowestCostTable successful in cost calculating.